google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreOpenTelemetryOptions.java

```diff
@@ -21,16 +21,16 @@
 import javax.annotation.Nullable;
 
 public class FirestoreOpenTelemetryOptions {
-  private final boolean enabled;
+  private final boolean tracingEnabled;
   private final @Nullable OpenTelemetry openTelemetry;
 
   FirestoreOpenTelemetryOptions(Builder builder) {
-    this.enabled = builder.enabled;
+    this.tracingEnabled = builder.tracingEnabled;
     this.openTelemetry = builder.openTelemetry;
   }
 
-  public boolean getEnabled() {
-    return enabled;
+  public boolean getTracingEnabled() {
```

Comment:
while we're here - I would rename this to `isTracingEnabled`

Reply:
Good point. Done.